### PR TITLE
Hpc - Update guide login to vSphere web interface from the control panel

### PR DESCRIPTION
**Pull request by pg-ovh**
Add guide update, links ans assets.

### CODE REVIEW

**Review by franckcosta — APPROVED** [review]
Good :)

**Comment by pg-ovh** [comment]
Hello @OVH-Team-Guides , let me known about proofread and I'll duplicates versions and add en-gb

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vsphere_interface_connexion/guide.fr-fr.md [hunk 1]
```diff
@@ -1,42 +1,121 @@
...
+![Connexion à l'interface vSphere HTML5](images/vsphere_web_client_local.png){.thumbnail}
 
 La page `Home`{.action} permet de retrouver les grands menus de votre vCenter.
 
+![Connexion à l'interface vSphere HTML5](images/vsphere_web_client_pcc_home.png){.thumbnail}
+
 ## Aller plus loin
 
-Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com>.
+Ce document de VMware répertorie les différents ports que vous devez ouvrir sur votre pare-feu pour, par exemple, accéder à la console : [Accès console](https://kb.vmware.com/kb/1012382).
```
Comment (Y0Coss) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vsphere_interface_connexion/guide.fr-fr.md]: @pg-ovh , I've moved this line at the bottom of the guide as it became unrelated to the part it was in. Feel free to remove it if it's not needed anymore.

**Review by Y0Coss — APPROVED** [review]

**Comment by Y0Coss** [comment]
Thank you for your contribution @pg-ovh

**Pull request merged**